<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sudoku</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="grid"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_newgame">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string>New Game</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_check">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 85, 255);</string>
          </property>
          <property name="text">
           <string>Check Game</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_reset">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 255, 255);</string>
          </property>
          <property name="text">
           <string>Reset Game</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_mode">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 127);</string>
      </property>
      <property name="text">
       <string>Dark Mode</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>447</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
